src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -691,6 +711,50 @@ SGMatrix<T> matrix_prod(SGMatrix<T>& A, SGMatrix<T>& B,
 }
 
 /**
+ * Performs the operation y = \alpha ax + \beta y
+ * This function multiplies a * x (after transposing a, if needed)
+ * and multiplies the resulting matrix by alpha. It then multiplies vector y by
+ * beta. It stores the sum of these two products in vector y
+ *
+ * @param scaling factor for vector ax
+ * @param vector a
+ * @param Whether to transpose matrix a
+ * @param vector x
+ * @param scaling factor for vector y
+ * @param vector y
+ */
+template <typename T>
+void cblas_dgemv(
```

Comment:
ok so let's just have it without the `cblas_` prefix as @lisitsyn suggested... so `dgemv`